tests/hap/test_svm_j97e06.py

```diff
@@ -0,0 +1,239 @@
+    filenames = list(table[filename_column])
+    print("\nread_csv_for_filenames. Filesnames from poller: {}".format(filenames))
+
+    return filenames
+
+
+@pytest.fixture(scope="module")
+def gather_data_for_processing(read_csv_for_filenames, tmp_path_factory):
+    # Create working directory specified for the test
+    curdir = tmp_path_factory.mktemp(os.path.basename(__file__)) 
+    os.chdir(curdir)
+
+    # Establish FLC/FLT lists and obtain the requested data
+    flc_flag = ""
+    flt_flag = ""
+    # In order to obtain individual FLC or FLT images from MAST (if the files are not reside on disk) which
+    # may be part of an ASN, use only IPPPSS with a wildcard.  The unwanted images have to be removed
+    # after-the-fact.
+    for fn in read_csv_for_filenames:
+        if fn.lower().endswith("flc.fits") and flc_flag == "":
+            flc_flag = fn[0:6] + "*"
+        elif fn.lower().endswith("flt.fits") and flt_flag == "":
+            flt_flag = fn[0:6] + "*"
+     
+        # If both flags have been set, then break out the loop early.  It may be
+        # that all files have to be checked which means the for loop continues
+        # until its natural completion.
+        if flc_flag and flt_flag:
+            break
+
+    # Get test data through astroquery - only retrieve the pipeline processed FLC and/or FLT files
+    # (e.g., j*_flc.fits) as necessary. The logic here and the above for loop is an attempt to
+    # avoid downloading too many images which are not needed for processing.
+    flcfiles = []
+    fltfiles = []
+    if flc_flag:
+        flcfiles = aqutils.retrieve_observation(flc_flag, suffix=["FLC"], product_type="pipeline")
+    if flt_flag:
+        fltfiles = aqutils.retrieve_observation(flt_flag, suffix=["FLT"], product_type="pipeline")
+
+    flcfiles.extend(fltfiles)
+
+    # Keep only the files which exist in BOTH lists for processing
+    files_to_process= set(read_csv_for_filenames).intersection(set(flcfiles))
+
+    # Identify unwanted files from the download list and remove from disk
+    files_to_remove = set(read_csv_for_filenames).symmetric_difference(set(flcfiles))
+    try:
+        for ftr in files_to_remove:
+           os.remove(ftr)
+    except Exception as x_cept:
+        print("")
+        print("Exception encountered: {}.".format(x_cept))
+        print("The file {} could not be deleted from disk. ".format(ftr))
+        print("Remove files which are not used for processing from disk manually.")
+
+    print("\ngather_data_for_processing. Gathered data: {}".format(files_to_process))
+
+    return files_to_process
+
+
+@pytest.fixture(scope="module")
+def gather_output_data(construct_manifest_filename):
+    # Determine the filenames of all the output files from the manifest
+    print("\nManifest Filename: {}".format(construct_manifest_filename))
+    files = []
+    with open(construct_manifest_filename, 'r') as fout:
+        for line in fout.readlines():
+            files.append(line)
```

Comment:
```suggestion
            files.append(line.rstrip("\n")
```
This should be needed in order for test code to work when using '.endswith(".fits")' (like on Line 184).